### DISCUSSION TITLE
Any way to only enable swipe back?

### DISCUSSION THREAD

**Discussion by erzads** [leandrowd/react-responsive-carousel#735] [Q&A]
I am looking for a way to only enable swiping back (to move forward the user must do a different action within the current slide).
I am almost at a point of trying to listen to the swipe events, calculate the direction based on the movement and so on but I am wondering if someone else already done this.

**Comment by medmor** [comment]
Hi, I have a similar use case when I want to swipe in one direction. Did find any solution for this problem.
